UnitTests/Tests/UnitTest1.cs

```diff
@@ -0,0 +1,144 @@
+namespace Tests;
+
+using System.Numerics;
+using UTest_NUnit;
+
+public class Tests
+{
+
+    [SetUp]
+    public void Setup()
+    {
+    }
+
+    [Test]
+    public void TestSumWithValidData()
```

Comment:
Suggest to change the name to TestSumWithPositiveNumbers (like you have below for Negative) as just Valid data is too general(e.i testing with negative numbers is still testing with valid data)